Sloth.Web/Controllers/BlobsController.cs

```diff
@@ -38,12 +38,15 @@ public class BlobsController : SuperController
         public async Task<IActionResult> Details(string id)
         {
             var blob = await DbContext.Blobs
+                .Where(b => b.Scrubbers.Any(s => s.Source.Team.Slug == TeamSlug)
+                    || b.TransactionBlobs.Any(tb => tb.Transaction.Source.Team.Slug == TeamSlug))
```

Comment:
Interesting case here -- since blob ID is a GUID I'm not sure this is needed, but i do like the extra security so if it works I think it's nice to have